helm/happa/templates/happaapi-ingress.yaml

```diff
@@ -1,12 +1,8 @@
-{{ if .Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress" }}
 apiVersion: networking.k8s.io/v1
-{{ else }}
-apiVersion: networking.k8s.io/v1beta1
-{{ end }}
 kind: Ingress
 metadata:
   name: happaapi
-  namespace: default
+  namespace: giantswarm
```

Comment:
Have you tested the namespace change thoroughly?

This ingress is a proxy to the Kubernetes API server (see `...backend.service.name: kubernetes`). I have tried moving the ingress resource to the giantswarm namespace in the past, and as a result, happaapi.* didn't work any more.

Reply:
No I didn't test this, but good point. I'll see what I can find!

Reply:
You're right! Reverted my changes.